# Motors
num_motors: 2

motors:
  motor_0:
    id: 1
    model: "XM430-W350"
    name: "joint_shoulder"
	ref_timeout: 0.2
  motor_1:
    id: 2
    model: "XM430-W350"
    name: "joint_elbow"
	ref_timeout: 0.2
